Bottom: 5519bea65aaa71062ba8c593f9da4a012d138da8
Top:    336be8fce9aa64b49656ef5fb327f195b89a3cac
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 22:22:57 +0000

Refresh of rename-resetfound-linesandpoly

---

diff --git a/src/set.c b/src/set.c
index f1b6c94..9fcc4dd 100644
--- a/src/set.c
+++ b/src/set.c
@@ -300,7 +300,7 @@ SetMode (int Mode)
       Crosshair.AttachedLine.State = STATE_FIRST;
       if (Mode == LINE_MODE && TEST_FLAG (AUTODRCFLAG, PCB))
 	{
-	  if (ResetConnections (true, FOUNDFLAG))
+	  if (ClearFlagOnAllObjects (true, FOUNDFLAG))
 	    {
 	      IncrementUndoSerialNumber ();
 	      Draw ();
